runtime/doc/api.txt

```diff
@@ -1416,8 +1416,9 @@ nvim_put({lines}, {type}, {after}, {follow})                      *nvim_put()*
                               • "l" |linewise| mode
                               • "" guess by contents, see |setreg()|
                     {after}   Insert after cursor (like |p|), or before (like
-                              |P|).
-                    {follow}  Place cursor at end of inserted text.
+                              |P|). Expects boolean value: true, or false.
+                    {follow}  Place cursor at end of inserted text. Expects
```

Comment:
"if true place cursor at end of inserted text." is enough. But you need to change it in api/vim.c to make permanent.

Reply:
Okay sounds good; that is less verbose :+1: . Thanks for the file navigation directions :+1: